### DISCUSSION TITLE
I'm not sure if this is a bug

### DISCUSSION THREAD

**Discussion by usuzumi-ii** [Jagailo/YetAnotherKeyDisplayer#34] [Q&A]
Sorry, my English is not very good, so I use a translator and only write short sentences.
When using Microsoft's Japanese input method: Using "shift + caps lock" will continuously display "D0, Oemtilde," and it cannot be dismissed. Then, using "alt + caps lock" will change it to "D1, Oemtilde." 
Using "ctrl + caps lock" will change it to "ShiftKey, OemBackslash," which, like before, will remain displayed and cannot be dismissed.
Thanks.

**Comment by Jagailo** [comment]
This may have been fixed in version 1.5. If not, please let me know.
